dataland-frontend/src/frameworks/gdv/ApiClient.ts

```diff
@@ -0,0 +1,35 @@
+// <--WARNING--> THIS FILE IS AUTO-GENERATED BY THE FRAMEWORK-TOOLBOX AND WILL BE OVERWRITTEN
```

Comment:
Do we now create ApiClient files that are framework specific? Up until now we had them (if i remember correctly) concentrated in one file. Or how does this here relate to ApiClients.ts and FrameworkApiClients.ts?

Reply:
Exactly. Previously I invested a lot of effort to put everything in one file. However, the resulting TS files are very hard to understand for anyone but the most experienced TS developers, and adding a new framework to this global api client requires changes to a buch of files (i.e., then the changes would not all be just in the `frameworks/gdv` directory) which was one of my design goals. But I'm happy to discuss alternatives.

Reply:
From my point of view this is fine. That would mean we would gradually get rid of our  global file every time we add an existing framework to the toolbox, right?